actioncable/lib/action_cable.rb

```diff
@@ -35,7 +35,8 @@ module ActionCable
       confirmation: 'confirm_subscription'.freeze,
       rejection: 'reject_subscription'.freeze
     },
-    default_mount_path: '/cable'.freeze
+    default_mount_path: '/cable'.freeze,
+    protocols: ["actioncable-v1-json".freeze].freeze
```

Comment:
The websocket on both ends takes an array and for potential backwards compatibility reasons, it may be necessary in the future for the server to be able to respond to multiple versions (cross your fingers this doesn't come true!).